handlers/deployments.go

```diff
@@ -0,0 +1,28 @@
+package handlers
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/kiali/kiali/services/business"
+	"net/http"
```

Comment:
just side note - these imports are un-organized. Kiali coding standard is:

import(
 ... go standard imports like net/http...
...blank line...
...third party imports like gorilla/mux...
...blank line...
...kiali imports
)